### DISCUSSION TITLE
[DEV] Dungeon Boss Skulls?

### DISCUSSION THREAD

**Discussion by ConcernedDev202X** [azerothcore/azerothcore-wotlk#3892] [General]
I appreciate the recent work by @Knindzagxg to clean up dungeons. However, I've noticed that all of these PRs also add skulls to all of the dungeon bosses. This isn't Blizz-like behavior and is noticeable by players as custom changes. I have looked for discussion on this but haven't found any. Is there a reason behind making these changes? I feel like it would be most appropriate for the skull flag to be saved for raid bosses, as is the case on official servers, unless there is an important reason behind the change.

An example PR that makes such a change is the recent #3306: https://github.com/azerothcore/azerothcore-wotlk/pull/3306

Thanks!

**Comment by Knindzagxg** [comment]
Its mostly visual fix.
Blizz intented and prepared everything for this long time ago during Vanilla.
Sadly, it never got official and fix never happened. There was quite few of those fixes which Blizz just forgot over time.
Things like this and even few more serious (Ulduar, Naxx and even ICC) are sorted trough Priv server devs.

When it comes to dungeons, its just visual.
Its big difference from raid bosses cause they always have higher level then current content (3 lvls higher are default) for scaling their damage with abilities or other complicated mechanics.

In dungeons level remain same and only skull is there instead.
You can see in official wowhead or any database they are classified as bosses cause they are and not just random NPCs.

That was one of the main reasons why we decided to add skull, to be as supposed.
Also, extra touch to gameplay is always good.

**Comment by Voxstrasza** [comment]
It's still not Blizzlike though and I thought that our main goal here was to make blizzlike content
